pom.xml

```diff
@@ -15,6 +15,18 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-core</artifactId>
```

Comment:
Isn't this dependency automatically imported? Doesn't s3 sdk depend on aws sdk already? Do we need to explicitly add this dependency?

Reply:
They were added to the pom file automatically. Might be able to remove